Mathlib/Algebra/Central/Matrix.lean

```diff
@@ -0,0 +1,38 @@
+/-
+Copyright (c) 2024 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang, Yunzhou Xie
+-/
+import Mathlib.Algebra.Central.Basic
+import Mathlib.Data.Matrix.Basis
+
+/-!
+# The matrix algebra is a central algebra
+-/
+
+namespace Algebra.IsCentral
+
+section MatrixAlgebra
```

Comment:
Why have this section given that there is only result in it?

Reply:
indeed